mithril-test-lab/mithril-end-to-end/src/mithril/aggregator.rs

```diff
@@ -48,6 +48,7 @@ impl Aggregator {
             ("GENESIS_SECRET_KEY", "5b3131382c3138342c3232342c3137332c3136302c3234312c36312c3134342c36342c39332c3130362c3232392c38332c3133342c3138392c34302c3138392c3231302c32352c3138342c3136302c3134312c3233372c32362c3136382c35342c3233392c3230342c3133392c3131392c31332c3139395d"),
             ("ERA_READER_ADAPTER_TYPE", "dummy"),
             ("ERA_READER_ADAPTER_PARAMS", &era_reader_adapter_params),
+            ("CARDANO_NODE_VERSION", "1.0.0"),
```

Comment:
Better to use the current version?
```suggestion
            ("CARDANO_NODE_VERSION", "8.1.2"),
```